### ISSUE TITLE
Not Authorized Error when Testing Admin Organization Search

### ISSUE THREAD

**Issue by vaidehijoshi** [thepracticaldev/dev.to#7824]
Related #4884 

```  
1) Admin manages organizations when searching for organizations searches for organizations
     Failure/Error: raise Pundit::NotAuthorizedError, "You must be logged in" unless user
     
     Pundit::NotAuthorizedError:
       You must be logged in
     
     
     
     # ./app/policies/application_policy.rb:5:in `initialize'
     # ./app/controllers/internal/application_controller.rb:12:in `authorize_admin'
     # ./spec/system/internal/admin_manages_organizations_spec.rb:12:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:117:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:117:in `block (2 levels) in <top (required)>'
Finished in 12 minutes 43 seconds (files took 7.32 seconds to load)
4750 examples, 1 failure, 3 pending
Failed examples:
rspec ./spec/system/internal/admin_manages_organizations_spec.rb:10 # Admin manages organizations when searching for organizations searches for organizations
```

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by mstruve** [comment]
Fixed: #8379
